packages/labs/virtualizer/src/test/layouts/flow.test.ts

```diff
@@ -54,7 +54,7 @@ describe('flow layout', () => {
     const virtualizer = (await until(() =>
       container.querySelector('lit-virtualizer')
     )) as LitVirtualizer;
-    expect(virtualizer).to.be.instanceof(LitVirtualizer);
+    await until(() => expect(virtualizer).to.be.instanceOf(LitVirtualizer));
```

Comment:
Is the `until()` use here for the instance check needed? Either the line above would resolve to the found `lit-virtualizer` element or would throw. I'm assuming it's not like we're waiting for a found `lit-virtualizer` element to get upgraded.

Unless you're thinking of combining them like
```ts
await until(() => expect(container.querySelector('lit-virtualizer')).to.be.instanceOf(LitVirtualizer));
```

Also TIL that chai aliases `.instanceof` and `.instanceOf`.

Reply:
Oh you know what I did leave that on accident.  I started converting those to handle a case that never happens and then I undid them but forgot and left that there.  Removed.